lunr/token_set_builder.py

```diff
@@ -44,15 +52,15 @@ def insert(self, word):
     def finish(self):
         self.minimize(0)
 
-    def minimize(self, down_to):
+    def minimize(self, down_to: int):
         for i in range(len(self.unchecked_nodes) - 1, down_to - 1, -1):
             node = self.unchecked_nodes[i]
             child_key = str(node["child"])
 
             if child_key in self.minimized_nodes:
                 node["parent"].edges[node["char"]] = self.minimized_nodes[child_key]
             else:
-                node["child"]._str = child_key
+                node["child"]._string = child_key
```

Comment:
Not totally clear what this attribute does, but it is `_string` in `TokenSet`, not `_str`.